project/profiles/capacity4more/modules/c4m/user/c4m_user_admin/c4m_user_admin.pages.inc

```diff
@@ -70,14 +70,12 @@ function c4m_user_admin_user_activate_form_submit($form, &$form_state) {
  * @see c4m_user_admin_user_activate_form()
  */
 function c4m_user_admin_user_reactivate_form_submit($form, &$form_state) {
-  c4m_user_admin_user_activate_form_submit($form, $form_state);
+  // Points that user is reactivated.
+  $form_state['user']->reactivate_submit = 1;
+  // Points if reactivate notification is required, or not.
+  $form_state['user']->reactivate_notification = $form_state['values']['c4m_notify_user'] ? 1 : 0;
```

Comment:
Actually, why do we need this ?
why not just using the `'c4m_notify_user'` on that _presave_ function ?